Jasmine/Common/ViewControllers/PhrasesExplorerViewController.swift

```diff
@@ -19,29 +36,37 @@ class PhrasesExplorerViewController: UIViewController {
     /// Injects the required data before opening this view.
     ///
     /// - Parameter viewModel: the view model of this class.
-    func segueWith(_ viewModel: PhrasesExplorerViewModel) {
+    func segueWith(_ viewModel: PhrasesExplorerViewModel, selectTableMode: Bool) {
```

Comment:
When the table is pressed, it can select (checkmark), it can segue to the Phrase view right. So this is true when the table is pressed